server/rpc_server.go

```diff
@@ -44,6 +46,7 @@ func newRpcServer(opts ...Option) Server {
 		handlers:    make(map[string]Handler),
 		subscribers: make(map[*subscriber][]broker.Subscriber),
 		exit:        make(chan chan error),
+		wg:          sync.WaitGroup{},
```

Comment:
It doesn't hurt, but this is not needed. The zero value of a `sync.WaitGroup` is the same as `sync.WaitGroup{}`.